README.md

```diff
@@ -77,6 +77,31 @@ include a separate stylesheet in order to support Internet Explorer
 
 [service-manual-browsers]: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in
 
+## Support for assistive technology
```

Comment:
Given we have a heading called 'Browser support' can we follow form and go with 'Assistive technology support'?